pages/account/index.js

```diff
@@ -0,0 +1,305 @@
+                  </Link>
+                </div>
+                <small className="text-muted">{ this.dateFormatter(order.created) }</small>
+              </td>
+              <td>
+                { this.getPaymentStatus(order.status_payment) }
+              </td>
+              <td>
+                { this.getFulfillmentStatus(order.status_fulfillment) }
+              </td>
+              <td>{ order.order_value.formatted_with_symbol }</td>
+              <td>
+                <Link href={`account/${order.id}`}>
+                  <a href={`account/${order.id}`} className="">View order</a>
+                </Link>
+              </td>
+            </tr>
+          )
+        })}
+      </tbody>
+    </table>
+    )
+  }
+
+  renderAlert() {
+    const { isError, message } = this.state;
+    if (!message) {
+      return null;
+    }
+
+    // Generate alert message as either list or single line.
+    const alertMessage = message.length === 1
+      ? message[0]
+      : (
+        <ul className="text-left m-0">
+          { message.map((copy) => <li key={copy}>{copy}</li>) }
+        </ul>
+      );
+
+    return (
+      <div className={`alert ${isError ? 'alert-danger' : 'alert-success'}`}>
+        { alertMessage }
+      </div>
+    );
+  }
+
+  render() {
+    // Displays message when the customer logs out.
+    if (!this.props.customer) {
+      return (
+        <Root>
+          <Head>
+            <title>commerce</title>
+          </Head>
+          <div className="account-container">
+            <div className="custom-container py-5 my-4 my-sm-5">
+              <div className="row mt-4">
+                <div className="col-12 text-center">
+                  <h2 className="font-size-header mb-4 pt-5">
+                    You have successfully logged out.
+                  </h2>
+                  <Link href="/">
+                    <a href="/" className="mt-4">Continue shopping</a>
+                  </Link>
+                </div>
+              </div>
+            </div>
+          </div>
+          <Footer />
+        </Root>
+      );
+    }
+    return (
+      <Root>
+        <Head>
+          <title>Account</title>
+        </Head>
+        <div className="account-container">
+          <div className="custom-container py-5 my-4 my-sm-5">
+            <div className="row mt-4">
+              <div className="col-12">
+                <h2 className="font-size-header mb-4 pt-5 text-center">
+                  My account
+                </h2>
+                { this.renderAlert() }
+              </div>
+            </div>
+            <div className="row mt-5 pt-5">
+              <div className="col-12 col-md-8 col-lg-8">
+                <div className="d-flex flex-row justify-content-between">
+                  <h5>Order history</h5>
+                  { this.customerSince() }
+                </div>
+                { this.ordersTable() }
```

Comment:
```suggestion
                { this.renderOrdersTable() }
```
I think that methods that render things in this repository are typically named `renderWhateverItRenders()`